### PR TITLE
Add profile flag

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Add profile flag

**Related issue(s)**

See https://github.com/kyma-project/cli/issues/640

### CODE REVIEW

File: pkg/installation/utils_test.go [hunk 1]
```diff
@@ -216,6 +216,53 @@ func Test_ReplaceDockerImageURL(t *testing.T) {
...
+	t.Parallel()
+	const replacedWithData = "evaluation"
+	testData := []struct {
+		testName       string
+		data           File
+		expectedResult File
+		shouldFail     bool
+	}{
+		{
+			testName: "correct data test",
```
Comment (clebs) [inline pkg/installation/utils_test.go:229]: Maybe we should also have a test case for incorrect data.

Comment (suleymanakbas91) [reply pkg/installation/utils_test.go:229]: Done.

File: pkg/installation/utils_test.go [hunk 2]
```diff
@@ -216,6 +216,62 @@ func Test_ReplaceDockerImageURL(t *testing.T) {
...
+			},
+			}},
+			shouldFail: true,
+		},
+	}
+
+	for _, tt := range testData {
+		err := insertProfile(&tt.data, replacedWithData)
+		if !tt.shouldFail {
+			require.Nil(t, err, tt.testName)
```
Comment (clebs) [inline pkg/installation/utils_test.go:267]:
```suggestion
			require.NoError(t, err, tt.testName)
```

File: pkg/installation/utils_test.go [hunk 3]
```diff
@@ -216,6 +216,62 @@ func Test_ReplaceDockerImageURL(t *testing.T) {
...
+			require.Equal(t, tt.data, tt.expectedResult, tt.testName)
+		} else {
+			require.NotNil(t, err, tt.testName)
```
Comment (clebs) [inline pkg/installation/utils_test.go:270]:
```suggestion
			require.Error(t, err, tt.testName)
```

**Review by clebs — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**